cypress/elements/fileMenu/index.js

```diff
@@ -1,43 +1,31 @@
-/*  FIXME:
-    Most of the element selection in this file is quite poor because 
-    of the FileMenu being old and still uses MUI.
-    This can be refactored to use proper data-test selectors once the new
-    FileMenu component is in place.
-*/
-
 import { clickMenuBarFileButton } from '../menuBar'
 
-const deleteModalEl = '*[class^="MuiDialog-container"]'
-const fileMenuItemEl = '*[role="menuitem"]'
+const deleteModalEl = '[data-test="file-menu-delete-modal"]'
```

Comment:
When you want to use a `data-test` prop, you just have to provide the name in the const and the use the `cy.getBySel` instead of `cy.get`.

Reply:
The same goes for all other `'[data-test="XYZ"]'` below

Reply:
Done.